corehq/apps/users/tests/test_dbaccessors.py

```diff
@@ -202,14 +203,25 @@ def usernames(users):
             [self.ccuser_inactive.username]
         )
 
+        filters = {'user_active_status': True}
```

Comment:
Thanks! 

(This once again shows that a test is only as good as it is written. If I included these cases originally this bug might not have happened.)